app/assets/stylesheets/mobile/mobile.scss

```diff
@@ -658,40 +658,62 @@ form#update_profile_form {
   .submit_block { margin-bottom: 20px; }
 }
 
-select#user_language, select#user_color_theme, #user_auto_follow_back_aspect_id, #aspect_ids_ {
-  padding: 3px;
-}
-
```

Comment:
I have added this again, except the aspect_ids_. A bad way to define styles for programmatic built Ids.. I found the f.select .. part which builds this Ids

Reply:
Not really the scope of this PR, but maybe it can be replaced by just a class then instead, which then doesn't need to be programmatically built and can be easier found? Or just select all `select` elements on the settings page (no idea why it's only these specific elements)? :thinking:

Reply:
I changed it to class settings. But I do not "see" any change when a select element has a padding.

Reply:
With the `padding: 3px`:
![image](https://user-images.githubusercontent.com/458548/188008868-03c3ffd6-54c0-4c7d-9363-e52936031fe4.png)
Without the `padding: 3px`:
![image](https://user-images.githubusercontent.com/458548/188008625-996d165c-b521-458d-8fcf-ac02d1b666f4.png)

Reply:
Ah, now I see it. Thanks.